src/lib/api/accounts.ts

```diff
@@ -0,0 +1,22 @@
+import type { Maybe, User, Org } from "@/api/types";
+import { BASE_API_URL } from "@/config/config.js";
+
+type Fetch = typeof globalThis.fetch;
+
+/** Get the logged-in user */
+export async function getMe(fetch: Fetch): Promise<Maybe<User>> {
+  const endpoint = new URL("users/me/", BASE_API_URL);
```

Comment:
I think we want to get the user's org here, by expanding the request. If a user switches orgs, we do that by sending a PATCH to `/api/users/me/`, and I think the return value will replace `$me`.

```suggestion
  const endpoint = new URL("users/me/?expand=organization", BASE_API_URL);
```

We may want to get all the user's organizations at this point, instead of just the active one, since that's not expandable.

Reply:
Sure, one request is better than two! The logic in `routes/+layout.ts` should not need to change if we do this.

We'll need organization names and avatars to display in the org switcher, so exanding all user orgs would be proactive.

Reply:
I updated the URL with the search param, and I wrote tests for the `getMe` that assert we're calling the endpoint as expected.